src/api/client.js

```diff
@@ -149,4 +149,23 @@ export class Client {
       }));
     },
   };
+
+  search = {
+    repos: async (q, params) => {
+      return this.call(`search/repositories?q=${q}`, params).then(response => ({
+        response,
+        nextPageUrl: this.getNextPageUrl(response),
+        schema: Schemas.REPO_ARRAY,
+        normalizrKey: 'items',
```

Comment:
This is the first use of `normalizrKey`.

The API response for this call looks like the following:

```
{
  total_count: 645354,
  incomplete_results: false,
  items: [
    { /* first repo object */},
    { /* second repo object */},
    ...
  ]
}
```

We only care for what's inside items, the array of repositories.

`normalizrKey` tells the dispatch proxy to [only treat that property](https://github.com/gitpoint/git-point/blob/master/src/api/proxies/create-dispatch-proxy.js#L105) of the JSON, instead of all the JSON.

Reply:
NICE ⭐️